plugins/analysis-icu/src/test/java/org/elasticsearch/index/mapper/CollationFieldTypeTests.java

```diff
@@ -42,8 +42,12 @@
 
     private static final Collator DEFAULT_COLLATOR = Collator.getInstance(ULocale.ROOT).freeze();
```

Comment:
Do we still need these changes?

Reply:
I think these are not strictly needed but don't hurt either. Mostly I got lazy while reverting all of the other changes :)